apps/spruce/tsconfig.json

```diff
@@ -1,38 +1,8 @@
 {
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "target": "esnext",
-    "lib": [
-      "dom",
-      "dom.iterable",
-      "esnext"
-    ],
-    "plugins": [
-      {
-        "name": "@styled/typescript-styled-plugin"
-      }
-    ],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "strict": false,
-    "forceConsistentCasingInFileNames": true,
-    "strictNullChecks": false,
-    "module": "CommonJS",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
-    "jsx": "react-jsx",
-    "jsxImportSource": "@emotion/react",
-    "types": [
-      "node",
-      "jest",
-      "@testing-library/jest-dom",
-      "new-relic-browser"
-    ],
     "baseUrl": "src",
-    "noFallthroughCasesInSwitch": true,
+    "strict": false
```

Comment:
I wonder if we should turn this on! unless it causes a bunch of type errors 🫣 then maybe we should make a ticket

Reply:
Ya there are mad errors 😳 opened DEVPROD-5887!